flexmeasures/ui/templates/accounts/account.html

```diff
@@ -228,7 +228,15 @@ <h3>Account</h3>
         </div>
       </div>
       <div class="card">
-        <h3 id="usersTableTitle">All users</h3>
+         <div class="d-flex justify-content-between align-items-center">
+          <h3 id="usersTableTitle">All users</h3>
+        
+          <a class="btn bg-primary-custom text-decoration-none text-light" data-bs-toggle="modal"
```

Comment:
We can wrap this link in `{% if user_can_create_children %}` (we do that below already for assets)